x/rollapp/types/chain_id.go

```diff
@@ -23,28 +23,35 @@ var (
 		regexEpoch))
 )
 
-// IsValidChainID returns false if the given chain identifier is incorrectly formatted.
-func IsValidChainID(chainID string) bool {
-	if len(chainID) > 48 {
-		return false
-	}
-
-	return ethermintChainID.MatchString(chainID)
+type ChainID struct {
+	ChainID  string
+	EIP155ID *big.Int
 }
```

Comment:
can we add a method to get the eip155id as uint64, rather than doing the conversion manually in loads of places

Reply:
done